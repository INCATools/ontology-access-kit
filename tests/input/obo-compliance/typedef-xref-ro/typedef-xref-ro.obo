format-version: 1.4
ontology: typedef-xref-ro

[Typedef]
id: R:1
xref: RO:0000000

