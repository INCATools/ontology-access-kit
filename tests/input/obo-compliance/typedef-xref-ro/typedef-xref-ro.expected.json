{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/typedef-xref-ro.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/RO_0000000",
      "type" : "PROPERTY",
      "meta" : {
        "xrefs" : [ {
          "val" : "RO:0000000"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasDbXref",
      "lbl" : "database_cross_reference",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}